spec/command/trunk/push_spec.rb

```diff
@@ -183,6 +183,13 @@ def found_podspec_among_files(files)
         cmd = Command.parse(%w(trunk push spec/fixtures/BananaLib.podspec --skip-tests))
         cmd.send(:validate_podspec)
       end
+
+      it 'passes use modular headers' do
```

Comment:
I wanted to make a test that matched https://github.com/CocoaPods/CocoaPods/blob/89a273882d75f42e51e486410dbab4fc15292a09/spec/functional/command/repo/push_spec.rb#L224-L236, but I had some issues trying to update the version of CocoaPods used with Bundler